### DISCUSSION TITLE
Publishing container

### DISCUSSION THREAD

**Discussion by ltomes** [ccutrer/waterfurnace_aurora#77] [General]
@ccutrer would I be allowed to publish a container containing this tool? I don't see a license so thought I would ask. If the answer is yes, do you have a preferred license for the published image?

Why?
The Dockerfile in this project has some out of date dependencies, if I fix those I want it all in CI so it's easy to bump versions in the future (and easy to keep "New Ruby broke" vs "Something in the waterfurnance gem broke" easy to detect.

**Comment by ccutrer** [comment]
You can do whatever you want - it's MIT licensed. I had referenced that in the gemspec, but neglected to add a proper LICENSE file. I've done so now.

Though obviously, I'd appreciate any fixes you make that you contribute back as PRs.

**Reply by ltomes** [reply]
I was pretty much just going to bump ruby and validate it works but you have already done that.
